docs/source/commands.rst

```diff
@@ -119,6 +119,40 @@ optional arguments:
 
     $ cfncluster delete mycluster
 
+ssh scp
+======
+
+Runs ssh or scp on the head node, with username and ip filled in based on the provided cluster.
+
+For example:
+    cfncluser ssh mycluster --args="-i ~/.ssh/id_rsa -v"
+
+Results in an ssh command with username and ip address pre-filled.
+    ssh -i ~/.ssh/id_rsa -v ec2-user@1.1.1.1
+
+Commands are defined in the global config file, under the aliases section and can be customized:
+
+    [aliases]
+    ssh = "ssh {ARGS} {CFN_USER}@{MASTER_IP}"
+    copy = "scp {CFN_USER}@{MASTER_IP}{ARGS}"
+
+Variables substituted:
+    {CFN_USER}
+    {MASTER_IP}
+    {ARGS} (only if specified on the cli)
+
+positional arguments:
+  cluster_name  name of the cluster to set variables for.
+
+optional arguments:
+  -h, --help    show this help message and exit
+  --dryrun, -d  print command and exit.
+
+::
+
+    $cfncluster ssh mycluster --args="-i ~/.ssh/id_rsa -v"

```

Comment:
do we have to use --args?  It would be more awesome if `cfncluster ssh mycluster -i ~/.ssh/id_rsa` just worked..

Reply:
I can do that. There's a function `parse_known_args()` that allows you to pass arbitrary args through, I can just append them to the command.